shared/middleware/files_api.rb

```diff
@@ -206,12 +250,44 @@ def put_file(endpoint, encrypted_channel_id, filename, body)
     put_file('assets', encrypted_channel_id, file[:filename], file[:tempfile].read)
   end
 
+  # POST /v3/animations/<channel-id>/<filename>?version=<version-id>
+  #
+  # Create or replace an animation. We use this method so that IE9 can still
+  # upload by posting to an iframe.
+  #
+  post %r{/v3/(animations)/([^/]+)/([^/]+)$} do |endpoint, encrypted_channel_id, filename|
```

Comment:
I also figured POST was the default method used by [the jQuery upload extension](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/assetManagement/AssetUploader.jsx#L16) we are using for asset uploads, for backwards-compatibility reasons.  I am reusing that plugin to get uploads up and running quickly for now.  I figure this has to be revisited at some point, as we'll probably be doing more client-side preprocessing of uploads in the future.